### PR TITLE
Improve UI

### LINKED ISSUE #98: Update UI to look better

### PR DESCRIPTION
**Pull request by Darkarche3**
Adds index for books.

### CODE REVIEW

File: src/main/java/seedu/address/ui/LibraryListPanel.java [hunk 1]
```diff
@@ -19,6 +21,10 @@ public class LibraryListPanel extends UiPart<Region> {
 
     @FXML
     private ListView<Book> libraryListView;
+    @FXML
+    private Label libraryTitle;
+    @FXML
+    private TitledPane pane;
```
Comment (meowwtama) [inline src/main/java/seedu/address/ui/LibraryListPanel.java:27]: Are these in use?

**Review by meowwtama — COMMENTED** [review]
Check usages of variables.

File: src/main/java/seedu/address/ui/LibraryListPanel.java [hunk 2]
```diff
@@ -4,8 +4,10 @@
 
 import javafx.collections.ObservableList;
 import javafx.fxml.FXML;
+import javafx.scene.control.Label;
```
Comment (meowwtama) [inline src/main/java/seedu/address/ui/LibraryListPanel.java:7]: Remove unused import.

File: src/main/java/seedu/address/ui/LibraryListPanel.java [hunk 3]
```diff
@@ -4,8 +4,10 @@
...
 import javafx.scene.control.ListCell;
 import javafx.scene.control.ListView;
+import javafx.scene.control.TitledPane;
```
Comment (meowwtama) [inline src/main/java/seedu/address/ui/LibraryListPanel.java:10]: Remove unused import.

**Review by meowwtama — COMMENTED** [review]
Remove the unused imports.

**Review by meowwtama — APPROVED** [review]
LGTM

**Review by Ty-stan0417 — COMMENTED** [review]
LGTM

**Pull request merged**